.php-cs-fixer.php

```diff
@@ -11,6 +11,9 @@
     ->in([
         __DIR__ . '/src',
         __DIR__ . '/tests',
+    ])
+    ->exclude([
+        'var/',
     ]);
 
 return $config
```

Comment:
In another context, we (as in teamneusta) agreed upon using the following rules to have the 'declare' on the same line as the php opening tag.
```
// declare strict types must be on first line after opening tag
        'blank_line_after_opening_tag' => false, // overwrite @Symfony
        'linebreak_after_opening_tag' => false, // overwrite @Symfony
        'declare_strict_types' => true, // custom
```

Reply:
This configuration is quite different from the other repos anyway.

Reply:
This might be a chance to reduce those differences :smile:

Reply:
Want to send a PR? :wink: